pages/Sword-of-Beheading.md

```diff
@@ -0,0 +1,13 @@
+The Sword of Beheading is a [Weapon](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Weapons) added in Slimefun.<br>
+It is an Iron Sword enchanted with Beheading II, which has a chance to drop a Player's or Mob's head upon killing them.
+
+## Chance to drop head
+The table belows lists the chance for each applicable entity to drop their head upon being killed.
+
+| Entity type | Drop chance |
+| ----------- | ----------- |
+| Player | 70% |
+| Creeper | 40% |
+| Skeleton | 40% |
+| Zombie | 40% |
+| Wither Skeleton | 25% |
```

Comment:
Maybe also add the following nice-to-know-thingy:

If you are using ExtraHeads, it also increases your chance to obtain heads from other mobs, see ExtraHeads.